### DISCUSSION TITLE
can't install the plugin in Calibre: "It does not contain a top-level __init__.py file"

### DISCUSSION THREAD

**Discussion by whiz4stars** [apprenticeharper/DeDRM_tools#2344] [Q&A] [answered]
i followed the instructions and tried to "Load plugin from file" with the "DeDRM_tools_10.0.9.zip" downloaded (I didn't unzip it).   The error message reads: "It does not contain a top-level __init__.py file"

does anyone have the same issue?  i searched but it seems some shared the same issue but no solution to it.  what can I do?

i use Windows 11 (64 bits) and in Calibre 7.16.0

**Comment by ElleKayEm** [comment]
Unzip DeDRM_tools_10.0.9.zip and load DeDRM_plugin.zip into calibre.

**Reply by whiz4stars** [reply]
as said, the plugin can't be loaded to calibre.  The error message reads: "It does not contain a top-level init.py file"

**Reply by ElleKayEm** [reply]
Then you aren't loading the correct file into calibre.  When you unzip DeDRM_tools_10.0.9.zip, you should see 5 files: the 2 plugin zip files and 3 readme text files.  Both of the plugin files can be loaded into calibre.  DeDRM is the one you probably want.  Obok is for books from Kobo Desktop or Kobo ereaders.

**Reply by whiz4stars** [reply]
i tried both the DeDRM_tools_10.0.9.zip and the DeDRM_plugin file (after unzipping it).

Whenever I tried to load the zip file to Calibre, the error message showed up.
![calibre_plugin_error](https://github.com/user-attachments/assets/c984a3b4-62e6-4e94-97ba-5383551f15de)

**Reply by ElleKayEm** [reply]
Only unzip DeDRM_tools_10.0.9.zip.  Do not unzip DeDRM_plugin.zip.  That will load into calibre.

**Reply by whiz4stars** [reply]
yes, I followed closely the instructions in the readme file.  I also tried all the possible ways (including what you mentioned) already but unfortunately, nothing works.

i tried zipped or unzipped DeDRM_tools_10.0.9.zip, and i tried the DeDRM_plugin.zip too.   the same error message "It does not contain a top-level __init__.py file" came up in whatsoever ways.

**Reply by ElleKayEm** [reply, marked as answer]
Okay, try this.  Delete the files you downloaded previously.  Download a fresh copy of DeDRM_tools_10.0.9.zip.  Leave it zipped.  Copy DeDRM_plugin.zip from inside DeDRM_tools_10.0.9.zip and paste it somewhere outside that's easy to navigate to like your desktop.  Load that copy of DeDRM_plugin.zip into calibre.

DeDRM_plugin.zip does contain _init_.py and does load into calibre.

**Reply by whiz4stars** [reply]
thanks a lot!  finally it worked!  gosh!
i unzip the DeDRM_tools_10.0.9.zip to a new folder on desktop and then load the DeDRM_plugin.zip to Calibre.
:)

**Reply by Mike-Lima** [reply]
ElleKayEm's answer worked for me as well. Thanks for that.

**Comment by Experimento28** [comment]
2026. aun funciona la solución de ElleKayEm
